src/plugins/simulator/simulations/brainwaves-gripper.ts

```diff
@@ -0,0 +1,24 @@
+export const kBrainwavesKey = "EMG_and_claw";
+
+export const brainwavesGripperSimulation = {
```

Comment:
To help make sure invalid edits to the simulation cause errors in this file you could add:
```suggestion
export const brainwavesGripperSimulation: ISimulation = {
```